contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -201,7 +202,33 @@ pltsql_createFunction(ParseState *pstate, PlannedStmt *pstmt, const char *queryS
 		return false;
 	}
 	else
-	{	
+	{
```

Comment:
do we use the same routine for both functions and procedures?

Reply:
Yes, we have a common structure `CreateFunctionStmt` for functions and procedures. In case of procedure, `CreateFunctionStmt->is_procedure` is true;